src/main/resources/templates/receiverTemplate.hbs

```diff
@@ -1,3 +1,10 @@
+{{!-- Helper function to handle status codes --}}
```

Comment:
Deze templates zijn wel heel erg cool en ik denk dat er best veel tijd bespaart kan worden met de juiste templates. Misschien leuk om eens te kijken naar de mogelijkheden om voor meer use-cases een template te bouwen en hoe dat dan in backstage geimplementeerd kan worden.